plan/physical_plan_test.go

```diff
@@ -379,6 +379,18 @@ func (s *testPlanSuite) TestDAGPlanBuilderJoin(c *C) {
 			sql:  "select /*+ TIDB_INLJ(t2) */ * from t t1 where t1.a in (select a from t t2)",
 			best: "MergeSemiJoin{TableReader(Table(t))->TableReader(Table(t))}(t1.a,t2.a)",
 		},
+		{
```

Comment:
add a test case that the join key is a int primary column

Reply:
Old test covered this.

Reply:
how about this one:
```sql
drop table if exists t1, t2;
create table t1(a bigint primary key, b bigint);
create table t2(a bigint, b bigint);
select /*+ TIDB_INLJ(t2) */ * from t1 join t2 on t1.a=t2.a and t1.b=t2.b;
```

Reply:
In the added case, a is int pk and f is another column.